it/test/controllers/StartPageControllerISpec.scala

```diff
@@ -160,6 +159,21 @@ class StartPageControllerISpec extends IntegrationTest with ViewHelpers {
       formPostLinkCheck(continueButtonHref, Selectors.continueButton)
     }
 
+    "render tailor return start page when the user is an individual and new tailoring feature switch is on" should {
```

Comment:
In this test how do we verify if it is the new page? may be we need to check redirect url?